app/tasks.py

```diff
@@ -404,41 +406,33 @@ def get(self):
                 logging.info("Timeout occurs in datastore")
                 self.add_task_for_repo(
                     self.repo, self.task_name(), self.ACTION)
-            finally:
-                self._notify(count_of_unreviewed_notes)
 
-    def _notify(self, count_of_unreviewed_notes):
+    def _maybe_notify(self, count_of_unreviewed_notes):
         # TODO: Response should be modified
-        if self._is_ok_to_notify(count_of_unreviewed_notes):
-            self.send_mail(self._get_admin_user_email(),
+        if self._should_modify(count_of_unreviewed_notes):
+            self.send_mail(self._get_unreviewed_notes_email(),
                            subject="PersonFinder: Please review your notes",
-                           body="{0} notes are not reviewed".format(
-                               count_of_unreviewed_notes))
+                           body="%d notes are not reviewed" %(
+                               count_of_unreviewed_notes,
+                           ))
             return True
         else:
             return False
 
-    def _is_ok_to_notify(self, count_of_unreviewed_notes):
-        email = self._get_admin_user_email()
-        if not email:
+    def _should_modify(self, count_of_unreviewed_notes):
+        email = self._get_unreviewed_notes_email()
+        if email == const.DEFAULT_UNREVIEWED_NOTES_EMAIL or not email:
             return False
 
-        thres = self._get_thres_unreviewed_notes()
-        if count_of_unreviewed_notes >= thres:
+        threshold = self._get_unreviewed_notes_email_threshold()
+        if count_of_unreviewed_notes >= threshold:
             return True
         else:
             return False
 
-    def _get_admin_user_email(self):
-        return self.config.get('admin_user_email')
-
-    def _get_thres_unreviewed_notes(self):
-        try:
-            return int(self.config.get('thres_unreviewed_notes'))
-        except (KeyError, ValueError, TypeError):
-            logging.info(
-                "'thres_unreviewed_notes' is not properly defined. " +
-                "Use DEFAULT_THRES_NUM_UNREVIEWED_NOTES({0}) instead".format(
-                    const.DEFAULT_THRES_NUM_UNREVIEWED_NOTES
-                ))
-        return const.DEFAULT_THRES_NUM_UNREVIEWED_NOTES
\ No newline at end of file
+    def _get_unreviewed_notes_email(self):
```

Comment:
Optional: You may remove this method. It is too simple; a method which calls just one method.